### PR TITLE
#U13 feat: implement  the expertise section on mobile breakpoints

### PR DESCRIPTION
**Pull request by Yvad60**
# PR Description

This PR creates the Expertise section for mobile devices (review on mobile)

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x] Created an array of expertise list
- [x] Add styles of the list on mobile

# Description of tasks that were expected to be completed and that are not yet complited
None
# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1.Checkout to the branch by running ` git checkout ft/expertise-section-mobile`
2. run `npm i` to install all required dependences 
3. run `npm run dev`

# Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case its appropriate and applicable

Screenshot 2022-11-15 160855
view on a 390px width mobile device

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by chrisostome01 — APPROVED** [review]
Well done @Yvad60

File: components/Sections/Expertise.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export default function Expertise() {
+  const expertiseList = [
```
Comment (IbrahimBagalwa) [inline components/Sections/Expertise.jsx]: you have to move expertise list and put it in the utils.

**Review by serapieTuyishime — APPROVED** [review]
Looking good!

File: tailwind.config.js [hunk 2]
```diff
@@ -1,14 +1,37 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    "./app/**/*.{js,ts,jsx,tsx}",
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
+    './app/**/*.{js,ts,jsx,tsx}',
+    './pages/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
   ],
```
Comment (IbrahimBagalwa) [inline tailwind.config.js:7]: change your singlecote to double

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
## Great job 👍 
just few changes

**Review by IbrahimBagalwa — APPROVED** [review]
## Great job 👍 

LGTM 💯

**Pull request merged**